code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -414,11 +430,11 @@
 
 	fix_modules()
 
-/obj/item/weapon/robot_module/peacekeeper
-	name = "peacekeeper robot module"
+/obj/item/weapon/robot_module/nations

```

Comment:
I really wish this had a better name than a meta reference to the name of the gametype.